products/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import AllLook, LookDetail
+
+urlpatterns = [
+    path('cruise-2019-20', AllLook.as_view()),
+    path('cruise-2019-20/<int:look_num>/', LookDetail.as_view()),
+]
```

Comment:
```suggestion
urlpatterns = [
    path('/cruise-2019-20', AllLook.as_view()),
    path('/cruise-2019-20/<int:look_num>', LookDetail.as_view()),
]
```

settings.py의 url을 변경하는 것과 더불어, 각 앱의 url에서는 앞에다 슬래시를 붙이는 것이 wecode coding convention 입니다 :)